wikimedia/wikipedia-ios
# get tyap working, update codemirror

**Fixes Phabricator ticket:** N/A

### Notes
* Gets Tyap working in the app.
* I misspoke in a meeting - the app permits translations even if there is not a wiki. (I was looking at another part of how we handle languages.) In this case, we had to follow step 3 of the [languages documentation](https://github.com/wikimedia/wikipedia-ios/blob/main/docs/languages.md), which I've updated in this PR to make even clearer.
* Most of this is codemirror stuff. The key pieces are the project file and `Wikipedia/iOS Native Localizations/kcg.lproj/InfoPlist.strings`.

### Test Steps
1. Add `Tyap` to a device (or sim's) preferred languages, and place it above English.
2. Run app, ensure you see Tyap.

### Screenshots
<img width="392" alt="Screenshot 2" src="https://user-images.githubusercontent.com/9295855/114794865-28a16400-9d42-11eb-93a5-c2778bd147de.png">
